tests/kafkatest/services/zookeeper.py

```diff
@@ -23,6 +23,7 @@
 
 
 class ZookeeperService(Service):
+    KAFKA_OPTS = ""
```

Comment:
I think this is a good point. It'd be nice to keep some unity across the services.